### PR TITLE
feat(filtering): support comments in the value files

### PR DESCRIPTION
**Pull request by Malinskiy**
Added support for comments in filter value files with `#` sign similar to https://github.com/MarathonLabs/marathon/pull/943

### CODE REVIEW

File: fixture/filtering/filetype_with_comments.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+filteringConfiguration:
```
Comment (matzuk) [inline fixture/filtering/filetype_with_comments.yaml:1]: where the comments?

File: fixture/filtering/tests_with_comments.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#line comment
```
Comment (matzuk) [inline fixture/filtering/tests_with_comments.txt:1]: I guess the comments from this file should be in the file above.

File: src/filtering/convert.rs [hunk 3]
```diff
@@ -423,6 +431,22 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn test_filetype_with_comments() -> Result<()> {
```
Comment (matzuk) [inline src/filtering/convert.rs:435]: also, it makes sense to write a test to check the file without comments too

Comment (Malinskiy) [reply src/filtering/convert.rs:435]: It already exists above

File: fixture/filtering/filetype_with_comments.yaml (hunk 1 above)
Comment (Malinskiy) [reply fixture/filtering/filetype_with_comments.yaml]: This is a regular YAML comment which should work out of the box, added

**Review by matzuk — APPROVED** [review]

**Pull request merged**